src/test/resources/uk/gov/ons/ctp/integration/rhsvc/endpoint/CaseEndpointUnitTest.CaseDTO.json

```diff
@@ -1,28 +1,28 @@
 [
 	{
 		"caseId": "dc4477d1-dd3f-4c69-b181-7ff725dc9fa4",
```

Comment:
for each CaseDTO consider adding other elements, and ensure tested in the corresponding unit test,

  - surveyId;
  - collectionExerciseId;
  - invalid;
  - refusalReceived;